### ISSUE TITLE
fail to reproduce the results of your experiment

### ISSUE THREAD

**Issue by Jiawen-huang** [kekmodel/FixMatch-pytorch#21]
I want to reproduce the result when the number of labeled data is 40
After executing the following command：
python train.py --dataset cifar10 --num-labeled 40 --arch wideresnet --batch-size 256 --lr 0.01 --expand-labels --seed 5 --out results/cifar10@40.5 --gpu 1

Accuracy hovered around 77.6, barely improving.
Is my hyperparameter set wrong?
How about your command?

Train Epoch: 148/1024. Iter: 1024/1024. LR: 0.0294. Data: 0.036s. Batch: 0.507s. Loss: 0.1614. Loss_x: 0.0009. Loss_u: 0.1605. Mask: 0.83. : 100%|█| 1024/
Test Iter:   79/  79. Data: 0.005s. Batch: 0.017s. Loss: 2.5019. top1: 77.20. top5: 96.61. : 100%|████████████████████████| 79/79 [00:01<00:00, 56.13it/s]
12/07/2020 18:27:21 - INFO - __main__ -   top-1 acc: 77.20
12/07/2020 18:27:21 - INFO - __main__ -   top-5 acc: 96.61
12/07/2020 18:27:21 - INFO - __main__ -   Best top-1 acc: 77.68
12/07/2020 18:27:21 - INFO - __main__ -   Mean top-1 acc: 77.20

**Comment by kekmodel** [comment]
I tested with --batch-size 64 --lr 0.03
